libs/providers/flagd/src/lib/service/http/service.ts

```diff
@@ -0,0 +1,155 @@
+import { ErrorCode, EvaluationContext, ResolutionDetails, StandardResolutionReasons } from "@openfeature/nodejs-sdk";
+import axios, { AxiosResponse, AxiosError } from "axios";
+import {
+  ResolveBooleanResponse,
+  ResolveNumberResponse,
+  ResolveStringResponse,
+  ResolveObjectResponse
+} from '../../../proto/ts/schema/v1/schema'
+import { Service } from "../Service";
+
+interface HTTPServiceOptions {
+  host: string;
+  port: number;
+}
+export class HTTPService implements Service {
+  private url: string
+
+  constructor(options: HTTPServiceOptions) {
+      this.url = `http://${options.host}:${options.port}`
```

Comment:
There is one small issue here - this locks us to http. We want to support HTTPS as well. Consider adding a protocol option or accepting a full URL.